quadlet/localstack/localstack.container

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=A Localstack container for local development of AWS services
+After=network-online.target
```

Comment:
Since the application requires the podman service, I think it should depend on it. So, I would add:
```
Requires=podman.socket
After=podman.socket
```
WDYT?

Reply:
Done!